src/Bicep.Cli.IntegrationTests/GenerateParamsCommandTests.cs

```diff
@@ -0,0 +1,347 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
Please (from the vscode only) open the updated/generated params file.

Reply:
done! thanks..